### ISSUE TITLE
Multiplayer crash when item tags modified with KubeJS

### ISSUE THREAD

**Issue by Xefyr0** [igentuman/NuclearCraft-Neoteric#125]
Hi - I'm making a modpack, and have a problem with Nuclearcraft: Neoteric crashing servers.

We have many mods with ingots and dusts like Gregtech and Thermal Expansion, but only want to use the ones from Gregtech. 
So, we use KubeJS to remove other mods' items from Forge item tags using this script:
```
ServerEvents.tags('item', event => {
    event.removeAllTagsFrom(/^(thermal|enderio|nuclearcraft):(powdered_.*|.*(_block|_plate|_ingot|_nugget|_gear|_dust))/)
})
```
Thermal Expansion and EnderIO have no problem, but Nuclearcraft: Neoteric crashes multiplayer servers. Singleplayer is unaffected.

Minecraft version: 1.20.1
Forge version: 47.3.7
KubeJS version: 2001.6.5-build.14
Nuclearcraft: Neoteric version: 1.0.0-rc.2

Crash log:
[message.txt](https://github.com/user-attachments/files/17485379/message.txt)

**Comment by igentuman** [comment]
This was fixed here: https://github.com/igentuman/NuclearCraft-Neoteric/commit/35ab30df8878b7980263d22a980110bcec23bb00

**Comment by Xefyr0** [comment]
> This was fixed here: [35ab30d](https://github.com/igentuman/NuclearCraft-Neoteric/commit/35ab30df8878b7980263d22a980110bcec23bb00)

EDIT: nevermind, didn't read things properly

Hope another release happens soon, can't wait!
